paddle/fluid/pybind/imperative.cc

```diff
@@ -824,6 +851,18 @@ void BindImperative(py::module *m_ptr) {
               return std::vector<int>();
             }
           })
+      .def_property_readonly(
+          "place",
```

Comment:
Better call VarBase.Place().

Reply:
x.shape x.stop_gradient x.persitaple x.place. Just keep consistent. 
torch suport tensor.device. 
May add place=Place() if needed in future. Such as grad=gradient().

Reply:
Sorry, what I really mean is that, the implementation of `place` already exists, which is the member function `VarBase::Place`.

Reply:
Done